### PR TITLE
fibjs 0.1.9

### PR DESCRIPTION
**Pull request by xicilion**

### CODE REVIEW

File: Library/Formula/fibjs.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Fibjs < Formula
   desc "JavaScript on Fiber"
   homepage "http://fibjs.org"
-  url "https://github.com/xicilion/fibjs/releases/download/v0.1.8/fibjs-0.1.8-fullsrc.zip"
-  sha256 "42ee9fa64a4259d88d648b8e5538226b2158a2d3546468b14a14b81c10ee8ed0"
+  url "https://github.com/xicilion/fibjs/releases/download/v0.1.9/fullsrc.zip"
```
Comment (DomT4) [inline Library/Formula/fibjs.rb]: You'll need to set an explicit `version` line here.

**Comment by apjanke** [comment]
New changes look good. (I tested locally on 10.9. (And holy cow does that take a while to build.))

Could you squash these to a single commit, so we have a tidy repo history and to make the PR easier to evaluate?

**Comment by DomT4** [comment]
It's good practice in general, but I'll squash locally here on pull to save the generally overworked CI another set of 12 minute builds.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/f9c67bb5f8012e9650e944e5310a6ee4502cbbb2. Thank you for another contribution to Homebrew @xicilion; we appreciate it! :smiley_cat:.

If you could squash in future per @apjanke's suggestion that'd make life a little easier for us.

**Comment by xicilion** [comment]
This is a huge project, including v8 JavaScript source and some private patch on it. Every upgrade may need to build full project and full v8 project. I don't know how to make it quick in this situation.

**Comment by DomT4** [comment]
Oh, no worries at all! The build time isn't a problem for the CI, I just didn't want to make it go through another build unnecessarily :)

**Pull request closed without merging**